tests/test_installation.py

```diff
@@ -0,0 +1,14 @@
+import platform
+import os
+
+
+def test_python_version():
```

Comment:
The test is OK, but instead of creating a new file, how about merging test_dependencies.py and test_installation.py into a single "more general" file?